docs/aerodromes/classc/EastSale.md

```diff
@@ -4,7 +4,149 @@
+  
+  - **Airborne**:  
+    If the pilot is already airborne and needs to switch to IFR, they must request a change of category as per the Local RTF Procedures. ATC will clear the aircraft as soon as practicable. If the clearance is not immediately available, ATC will instruct the pilot to "REMAIN VFR" and provide necessary control instructions or expectations.
+  
+  - **IFR Climb**:  
+    On departure from ESL or WSL, a pilot may request an IFR climb to penetrate weather en route to a training area. During the day, unless being vectored, the pilot is responsible for terrain clearance below LSALT. Aircraft cleared for an "IFR CLIMB" must continue climbing, not above their cleared level, and report "CANCEL IFR" when able to resume VFR.
+  
+- **Resuming VFR**:
+  - Recovering military aircraft must automatically revert to VFR at the following points:
+    - At the initial point when recovering via military stream landing pattern (initial and pitch).
+    - At Hi-Key.
+    - Following a touch and go, go-around, or visual overshoot when a local IFR aircraft has indicated an intention to join the circuit.
+
+## Runway Modes
+Single runway operations only.
+
+## Circuit Procedures
+
+Circuit operations occur within a `5NM` radius of ESL ARP, at the following altitudes:
+
+<table>
+  <thead>
+    <tr>
+      <th>Type</th>
+      <th>Normal Altitude</th>
+      <th colspan="2">Low Level Altitude</th>
+    </tr>
+    <tr>
+      <th></th>
+      <th></th>
+      <th>Day</th>
+      <th>Night</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Non Jet</td>
+      <td>A010</td>
+      <td>A004</td>
+      <td>Circling Minima</td>
+    </tr>
+    <tr>
+      <td>Jet</td>
+      <td>A015</td>
+      <td>A006</td>
+      <td>Circling Minima</td>
+    </tr>
+  </tbody>
+</table>
+
+ATC should avoid the use of `MAINTAIN/EXTEND DOWNWIND` except when operationally required for separation with IFR aircraft or sequencing crossing runway traffic.
+
+### Circuit Direction
+| Runway | Direction |
+| ------ | ----------|
+| 09     | Left      |
+| 22     | Left      |
+| 04     | Right     |
+| 27     | Right     |
+
+## Circuit Saturation
+ATC should declare the circuit saturated when aircraft exceeds the below table.
+When the circuit is saturated further circuits will be unavailable until numbers are within tolerance of the table.
+This number may be reduced by ATC due to complexity of operations and prevailing weather in order to ensure safety.
+
+| Time   | ESL Day | ESL Night | WSL Day | WSL Night |
+| ------ | ------- | --------- | ------- | --------- |
+| Day    | 6       | 5         | 5       | 5         |
+
+## Low Approach
+- By day, pilots of local aircraft may request a low approach (descent to 50FT AGL) when less than the applicable runway separation standard exists.
```

Comment:
Can you please define how this assists with runway separation? Otherwise, just reduce this section to read "Pilots may request a low approach"